hunting/windows/queries/execution_via_windows_management_instrumentation_by_occurrence_frequency_by_unique_agent.toml

```diff
@@ -0,0 +1,45 @@
+[hunt]
+author = "Elastic"
+description = """
+This hunt looks for unique process execution via Windows Management Instrumentation (WMI) by removing random patterns from `process.command_line` and aggregating execution by count of agents with the same command line to limit results to unique ones.
+"""
+integration = ["endpoint", "windows", "system"]
+uuid = "b5efeb92-9b51-45b9-839f-be4cdc054ef4"
+name = "Low Frequency of Process Execution via WMI by Unique Agent"
+language = "ES|QL"
+license = "Elastic License v2"
+notes = [
+    "This hunt contains three queries for Elastic Defend, Sysmon, and Windows Security event 4688."]
+mitre = [ "T1047"]
+query = [
+'''
+from logs-endpoint.events.process-*, logs-windows.sysmon_operational-*
+| where @timestamp > now() - 7 day and
+  host.os.family == "windows" and event.category == "process" and event.action in ("start", "Process creation") and
+  process.parent.name.caseless == "wmiprvse.exe" and starts_with(process.code_signature.subject_name, "Microsoft")
```

Comment:
I don't think these would work as only Elastic Defend has caseless fields atm

Reply:
using `to_lower` which was added in 8.14